apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ConsumerController.java

```diff
@@ -119,6 +120,15 @@ public List<ConsumerRole> assignNamespaceRoleToConsumer(@PathVariable String tok
     return consumerService.assignNamespaceRoleToConsumer(token, appId, namespaceName);
   }
 
+  @GetMapping("/consumers/list")
+  @PreAuthorize(value = "@permissionValidator.isSuperAdmin()")
+  public List<Consumer> getConsumerList(Pageable page){
+    return consumerService.findAllConsumer(page);
+  }
 
+  @DeleteMapping(value = "/consumers/delete/by-appId")
+  public void deleteConsumers(@RequestParam String appId) {
```

Comment:
I think we need to also check the permission for this method?

Reply:
yes i missed it